bigquery/google/cloud/bigquery/client.py

```diff
@@ -284,7 +284,7 @@ def list_datasets(
             filter (str):
                 Optional. An expression for filtering the results by label.
                 For syntax, see
-                https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets/list#filter.
+                https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets/list#body.QUERY_PARAMETERS.filter.
```

Comment:
Let's remove the trailing `.` to make sure the link works as expected.
```suggestion
                https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets/list#body.QUERY_PARAMETERS.filter
```